content/awell-orchestration/api-reference/webhooks/form-submitted.mdx

```diff
@@ -22,6 +22,19 @@ showPlaygroundButton: false
     },
     {...}
   ],
+  "formData": {
```

Comment:
@rahulkeerthi Outside-in perspective:

[1] It's forcing me to think about what the difference is between `response` and `formData`. Do both contain the actual response values, do they both contain data? `response` and `form` (or `form_definition`) would have been better maybe.

[2] We are mixing different types of casing (I can see that was already the case previously). It's one of these things a customer will look at and ask themselves: why?? @bejoinka @ebomcke-awell especially for our API, we need to agree on a basic principle for what type of casing we use moving forward. Can you pick that up?